spec/controllers/idv/sessions_controller_spec.rb

```diff
@@ -69,18 +69,26 @@
       end
     end
 
-    context 'with pending in person enrollment' do
+    context 'with in person enrollment' do
       let(:user) { build(:user, :with_pending_in_person_enrollment) }
 
       before do
         allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
       end
 
-      it 'cancels in person enrollment' do
+      it 'cancels pending in person enrollment' do
         delete :destroy
 
         expect(user.reload.pending_in_person_enrollment).to be_blank
       end
+
+      it 'cancels establishing in person enrollment' do
+        create(:in_person_enrollment, :establishing, user: user)
+        expect(InPersonEnrollment.where(user: user, status: :establishing).count).to eq(1)
+        delete :destroy
+
+        expect(InPersonEnrollment.where(user: user, status: :establishing).count).to eq(0)
+      end
```

Comment:
Similar comment here, i.e. actually make sure the enrollment is cancelled in addition to this check.

Reply:
Updated in 4312bd866